packages/cli/tests/unit/record/record.test.spec.ts

```diff
@@ -67,6 +79,64 @@ describe('record test', () => {
     });
   });
 
+  describe('record test with invalid directory parameter', () => {
+    it('shows error if directory parameter is file instead of directory', async () => {
+      const dirPrefix = tmpdir();
+      const directoryParam = `${dirPrefix}/file_not_dir`;
+      // ensuring the file doesn't exist breaks after trying it a few times
+      // await unlink(directoryParam, (err) => {});
+      closeSync(openSync(directoryParam, 'w'));
+      expect(existsSync(directoryParam)).toEqual(true);
+
+      let argv = {
+        _: ['record', 'test'],
+        $0: 'src/cli.ts',
+        directory: directoryParam,
+        d: dirPrefix,
+      };
+
+      const ret = await RecordCommand.handler(argv);
+      expect(ret).toEqual(null);
+
+      await unlink(directoryParam, (err) => {});
+    });
+
+    it('shows error if directory parameter is invalid symlink', async () => {
```

Comment:
Same as above, this is a little misleading since you're not actually testing the output to the terminal.